feat(openapi-ts): export additional functions from openapi-ts

seriouslag opened the pull request:
Added exports for getSpec, initConfigs, and parseOpenApiSpec to the new exported internal file for improved accessibility of these functions across internal packages

These functions are helpful for custom clients and plugins.

Marked the exported functions as @internal, and I put them in a new export path

mrlubos commented on packages/openapi-ts/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hey-api/openapi-ts",
-  "version": "0.66.3",
+  "version": "0.67.0",
```
Can you revert this change? Not sure how that happened

seriouslag replied:
reverted

seriouslag committed:
feat(openapi-ts): export additional functions from index.ts

Added exports for getSpec, initConfigs, and parseOpenApiSpec to a new internal export file for improved accessibility of these functions for non public consumption.

Marked these functions as internal as they are not to be used publicly

Commit:
Create three-rats-count.md
